.github/actions/setup-nvidia/action.yml

```
@@ -140,11 +157,36 @@ runs:
               )
           }
 
+          install_nvidia_driver_amzn2() {
+              (
+                  set -x
+
+                  pre_install_nvidia_driver_amzn2
+
+                  install_nvidia_driver_common
+
+                  post_install_nvidia_driver_common
```

Not sure why do you need an empty line between the statements here. What do they symbolize?
```suggestion
                  set -x
                  pre_install_nvidia_driver_amzn2
                  install_nvidia_driver_common
                  post_install_nvidia_driver_common
```